include/seqan3/alignment/pairwise/detail/policy_optimum_tracker.hpp

```diff
@@ -86,7 +86,7 @@ struct max_score_updater
  */
 template <typename alignment_configuration_t, std::semiregular binary_updater_t>
 //!\cond
-requires is_type_specialisation_of_v<alignment_configuration_t, configuration>
+    requires is_type_specialisation_of_v<alignment_configuration_t, configuration>
```

Comment:
Well you can just squash this commit ([NOP] Fix indentation in policy optimum tracker. ) into the former ([MISC] Introduce binary matrix updater..)

Reply:
Why? It has nothing to do with the former commit.